src/components/itemBox/ItemBox.tsx

```diff
@@ -48,11 +74,12 @@ export class ItemBox extends React.Component<IItemBoxProps, any> {
         return (
             <Tooltip {...this.props.tooltip}>
                 <li
+                    ref={(li) => this.el = li}
                     className={this.getClasses()}
                     onClick={() => this.handleOnOptionClick()}
                     data-value={this.props.value}>
                     {this.props.prepend ? <Content {...this.props.prepend} /> : null}
-                    {this.props.displayValue || this.props.value}
+                    <PartialStringMatch wholeString={this.props.displayValue || this.props.value} partialMatch={this.props.highlight} caseInsensitive />

```

Comment:
I wonder if `caseInsensitive={true}` should not be the default in PartialStringMatch 🤔